waiter/bin/ci/saml-idp-server-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# Run SAML idendity provider (IdP) server via docker
+echo Starting SAML IdP server docker container
+docker run --name=testsamlidp_idp --detach --rm -p 8090:8090 -p 8443:8443 \
+-e SIMPLESAMLPHP_SP_ENTITY_ID=waiter \
+-e SIMPLESAMLPHP_SP_ASSERTION_CONSUMER_SERVICE=http://localhost:9091/waiter-auth/saml/acs \
+-e SIMPLESAMLPHP_SP_SINGLE_LOGOUT_SERVICE=http://localhost:9091/waiter-auth/saml/logout \
+-d kristophjunge/test-saml-idp:1.15
```

Comment:
Should we craft a hand-written server that responds with the status/headers/body we like. We did with the data local server in Cook.

Reply:
currently the docker image works fine without special case handling